acceptance/bundle/artifacts/artifact_upload_for_volumes/.gitignore

```diff
@@ -0,0 +1 @@
+*.whl
```

Comment:
what's the purpose of this .gitignore, how does it affect this test?

Why does it upload anything? Because libraries setting overrides gitignore?

Reply:
Left over from the other iteration, we indeed don't need it here